### PR TITLE
DIFM: Replaces Sitemap with Testimonials on the Page Selector Step

### PR DESCRIPTION
**Pull request by aneeshd16**
#### Changes proposed in this Pull Request
p1653050432966909-slack-C02KVCAL7GX
* Replaces the Sitemap page with Testimonials page on the Page Selector Step.
* Changed the colour of the thumbnails to increase contrast. p1653075691656219/1653050432.966909-slack-C02KVCAL7GX

|Before|After|
|---|---|
|image|image|

#### Testing instructions

* Go to `/start/do-it-for-me`
* On the Page Selector step, confirm that you can see the Testimonials page. (Please see the screenshot above)
* Select the Testimonials page and complete the product purchase.
* On the Website Content step, confirm that you can see the Testimonials page.
image

### CODE REVIEW

File: client/signup/difm/components/BrowserView.tsx [hunk 1]
```diff
@@ -55,26 +57,26 @@ const Container = styled.div< { isSelected?: boolean; isClickDisabled?: boolean
...
-    padding: 5px 0 0px 6.5px;
+	height: 7px;
+	position: relative;
+	border-radius: 6px;
+	padding: 5px 0 0px 6.5px;
 	img {
-    	position: absolute;
+		position: absolute;
 	}
-}`;
```
Comment (aneeshd16) [inline client/signup/difm/components/BrowserView.tsx:65]: Removed extra bracket.

File: client/signup/difm/components/BrowserView.tsx [hunk 2]
```diff
@@ -55,26 +57,26 @@ const Container = styled.div< { isSelected?: boolean; isClickDisabled?: boolean
...
+	position: absolute;
 	width: ${ ( { isSelected } ) => ( isSelected ? '224px' : '222px' ) };
 	left: ${ ( { isSelected } ) => ( isSelected ? '-1px' : '0' ) };
-    top: 12px;
-	&:hover{
+	top: 12px;
+	&:hover {
 		width: ${ ( { isSelected } ) => ( isSelected ? '224px' : '221px' ) };
 	}
-}`;
```
Comment (aneeshd16) [inline client/signup/difm/components/BrowserView.tsx:77]: Same as above.

**Comment by jdc91** [comment]
LGTM 🚢 
The colour could have been changed with a [filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter). 
(Colour can be generated from https://angel-rs.github.io/css-color-filter-generator/.)
But since this is already done it does not matter 👍 Just sharing a personal learning with you in case you find it useful :)

**Review by ddc22 — APPROVED** [review]
LGTM 🚢
The colour could have been changed with a [filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter).
(Colour can be generated from https://angel-rs.github.io/css-color-filter-generator/.)
But since this is already done it does not matter 👍 Just sharing a personal learning with you in case you find it useful :)

**Pull request merged**